components/ILIAS/Authentication/classes/Password/class.LocalUserPasswordManager.php

```diff
@@ -18,21 +18,31 @@
 
 declare(strict_types=1);
 
-class ilUserPasswordManager
+namespace ILIAS\Authentication\Password;
+
+use ilAuthUtils;
+use ilDBInterface;
+use ilFileUtils;
+use ilObjUser;
+use ilPasswordException;
+use ilPasswordUtils;
+use ilSession;
+use ilSetting;
+use ilUserException;
+
+class LocalUserPasswordManager
 {
     private const MIN_SALT_SIZE = 16;
-
```

Comment:
Please re-introduce the empty lines as an optical separator between constants, static properties and members.